ktor-features/ktor-metrics-micrometer/api/ktor-metrics-micrometer.api

```diff
@@ -1,18 +1,18 @@
 public final class io/ktor/metrics/micrometer/MicrometerMetrics {
 	public static final field Feature Lio/ktor/metrics/micrometer/MicrometerMetrics$Feature;
-	public static final field activeGaugeName Ljava/lang/String;
```

Comment:
This change is a problem. We can't have any removals in non-major versions. You need to keep old constants, but deprecate them and have some other meaningful name for new properties.